src/Exception/Binary/Loader/ChainAttemptNotLoadableException.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Exception\Binary\Loader;
+
+use Liip\ImagineBundle\Binary\Loader\LoaderInterface;
+
+class ChainAttemptNotLoadableException extends NotLoadableException
+{
+    private string $configName;
+    private LoaderInterface $objectInst;
+
+    public function __construct(string $configName, LoaderInterface $objectInst, NotLoadableException $loaderException)
```

Comment:
@franmomu @dbu Are you both comfortable with changing `objectInst` to `loaderInst` (instead of the suggested variable name of `loader`) with regard to the *exception class __only__*? Specifically:

- `ChainAttemptNotLoadableException::$objectInst` *->* __the original property name__
- `ChainAttemptNotLoadableException::$loaderInst` *->* __my suggested property name__

This better aligns with the methods as they are currently defined for the chain-attempt exception, IMHO. For example:

- `ChainAttemptNotLoadableException::getLoaderConfigName()`
- `ChainAttemptNotLoadableException::getLoaderObjectInst()`
- `ChainAttemptNotLoadableException::getLoaderObjectName()`

*(Not that it would matter to anyone else, but my suggestion also better satisfies my OCD which prefers same-length groups of properties and methods, when feasible to do so without convoluting their meaning...)*

Let me know if this is objectionable to either of you; I'll perform the original change if you both are not agreeable to my suggested alternative.